manifests/x/XiamenYajiSoftwareCoLtd/CocosDashboard/1.2.1.2819/XiamenYajiSoftwareCoLtd.CocosDashboard.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: XiamenYajiSoftwareCoLtd.CocosDashboard

```

```suggestion
PackageIdentifier: XiamenYaji.Cocos.Dashboard
```
Package exists under a different identifier. Please use the existing identifier for updates